scio-core/src/main/scala/com/spotify/scio/values/PairSkewedSCollectionFunctions.scala

```diff
@@ -56,6 +56,73 @@ object HotKeyMethod {
 
 }
 
+object SkewedJoins {
+
+  // some sensible defaults for skewed joins
+  val DefaultHotKeyThreshold: Int = 9000
+  val DefaultHotKeyMethod: HotKeyMethod.Threshold = HotKeyMethod.Threshold(DefaultHotKeyThreshold)
+  val DefaultHotKeyFanout: Int = 5
```

Comment:
how was 5 chosen? Any change it will degrade performance if not overridden?

Reply:
Set to no fanout. See comment